charts/images.yaml

```diff
@@ -2,8 +2,8 @@ images:
 - name: etcd-backup-restore
   sourceRepository: github.com/gardener/etcd-backup-restore
   repository: eu.gcr.io/gardener-project/gardener/etcdbrctl
-  tag: "v0.14.0"
+  tag: "v0.15.0"
```

Comment:
Same as above

Reply:
Why? This value should be updated, right? IMHO, We are not expecting this release of Druid to work with `v0.14.0` of `backup-restore`.

Reply:
Simply because:
a) There will be a separate PR for `v0.15.0`.
b) `v0.15.0` is not released yet.
c) If we want to merge this PR before, the `main` branch is for sure broken (mainly for single node etcd).

Reply:
Done